app/settings.py

```diff
@@ -108,7 +81,7 @@ class Config:
             },
         },
         "loggers": {
-            "": {
+            "logger": {
```

Comment:
Hmm, `""` configures the root logger, "logger" configures a logger called "logger", which I'm not sure is something we actually use?

Reply:
Ah sorry, I forgot to delete that while I was doing some testing